playbooks/conferences.md

```diff
@@ -24,6 +24,30 @@ guidelines:
 - If you need help preparing a conference talk, stop by [Ash][ash]'s weekly Writing Office Hours.
 - It's also recommended sharing the talk and lessons with the team.
 
+## Approval Process
+
+- Find an event (conference/seminar/course) that you would like to attend. Identify items in the agenda that will
+  contribute to both your growth goals and Artsy's business goals.
+- Estimate any travel or lodging expenses that may be needed to attend the event using Concur to search for airfare
+  and hotels.
+- Speak to your manager about attending the event before purchasing tickets. Communicate the following items:
+  - How will this event contribute to your and your team's goals?
+  - What are the dates of the event?
+  - What is the attendance fee for the event?
+  - What travel and lodging feens will be required?
+  - Managers should consider the following:
```

Comment:
I'm glad we've got these listed, too!

Reply:
ya +1! seeing this list, I'm wondering if there are any more formal requirements we should mention/link to here. Is there a place for managers to view the amount of the event budget that remains? Or for them to see how many engineers have attended conferences? Is there some kind of #engineering-managers channel in Slack they could post in to ask these types of questions?

None of this is a hard requirement, to be sure, just considering what I would want if I were a manager doing this for the first time :)